hail/python/hail/dev-environment-py3.7.yml

```diff
@@ -14,9 +14,10 @@ dependencies:
 - pip=18.0
 - pytest=3.9.2
 - pytest-html=1.19
+- flask=0.12.2
```

Comment:
flask shouldn't be in the hail dev environment, we don't use it in hail.

We should probably get rid of these files anyway, we're just using the root requirements.txt now

Reply:
Will do